packages/mdc-linear-progress/foundation.ts

```diff
@@ -35,30 +36,32 @@ export default class MDCLinearProgressFoundation extends MDCFoundation {
     return strings;
   }
 
-  static get defaultAdapter() {
+  static get defaultAdapter(): MDCLinearProgressAdapter {
+    /* tslint:disable:no-empty */
     return {
-      addClass: (/* className: string */) => {},
-      getPrimaryBar: () => /* el: Element */ {},
-      getBuffer: () => /* el: Element */ {},
-      hasClass: (/* className: string */) => false,
-      removeClass: (/* className: string */) => {},
-      setStyle: (/* el: Element, styleProperty: string, value: string */) => {},
+      addClass: (_className: string) => {},
+      getBuffer: () => null,
+      getPrimaryBar: () => null,
+      hasClass: (_className: string) => false,
+      removeClass: (_className: string) => {},
+      setStyle: (_el: HTMLElement, _styleProperty: string, _value: string) => {},
```

Comment:
I don't think we can't use `keyof CSSStyleDeclaration` here because it lacks vendor prefixes:

https://github.com/material-components/material-components-web/blob/73ae286beb6305183c8d0ee06226ebd382c776e9/packages/mdc-animation/index.ts#L69

https://github.com/material-components/material-components-web/blob/3a874f554895b40e1bafd5937ab9b4fcb3b8d756/packages/mdc-linear-progress/foundation.ts#L106-L108

https://github.com/material-components/material-components-web/blob/3a874f554895b40e1bafd5937ab9b4fcb3b8d756/packages/mdc-linear-progress/index.ts#L65

I'm pretty sure we can get rid of `transformStyleProperties` because all major browsers now implement `transform` either unprefixed or `-webkit-`prefixed, according to [Can I Use](https://caniuse.com/#feat=transforms2d).

Even so, `getCorrectPropertyName()` returns `StandardCssPropertyName | PrefixedCssPropertyName`, which is not directly type-compatible with
`keyof CSSStyleDeclaration`.

WDYT?